src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,37 @@
+.card__rating {
```

Comment:
It's an element of `card` block, so it shouldn't be in this file. Move into the file with card styles